calico-cloud/visibility/packetcapture.mdx

```diff
@@ -48,11 +48,37 @@ For a simple use case workflow see, [Faster troubleshooting of microservices, co
 
 ## How To
 
+- [Enable Packet capture](#enable-packet-capture)
 - [Packet capture in Service Graph](#packet-capture-in-service-graph)
 - [Packet capture using the command line](#packet-capture-using-the-command-line)
 - [Store and rotate capture files](#store-and-rotate-capture-files)
 - [Enforce RBAC for capture tasks for CLI users](#enforce-rbac-for-capture-tasks-for-cli-users)
 
+### Enable Packet capture
+
+##### Enable PacketCapture using kubectl
+
+Create a PacketCaptureAPI custom resource, named `tigera-secure` in the cluster.
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: operator.tigera.io/v1
+kind: PacketCaptureAPI
+metadata:
+  name: tigera-secure
+EOF
+```
+
+##### Enable PacketCapture using Manager UI
+
+On the Manager UI, click **Service Graph**, select an endpoint from the service graph, right-click, and select **Initiate packet capture**. If the packet capture feature is not enabled, click enable to activate the packet capture feature.
```

Comment:
This feels like a lot for a single step. Perhaps:

1. From the Manager UI, click the **Service Graph** to view the service graph diagram for your cluster.
2. Right-click on any of the endpoints and select **Initiate packet capture**. 
3. On the **NAME HERE** dialog, click **Enable** to activate the packet capture feature.

Reply:
This seems like a backdoor method of enabling a feature. Is there not a more direct way to do this?

Also, will this enable packet capture for the whole cluster, or just the endpoint?

Reply:
This would enable packetcapture for entire cluster, Previously we had this enabled by default hence in order to not to divert much from the functionality/rework. Phil suggest this UI flow.

Reply:
Fair enough. Let's reorganize the procedure as I outlined above. Somethin like:

1. From the Manager UI, click the Service Graph to view the service graph diagram for your cluster.
2. Right-click on any of the endpoints and select Initiate packet capture.
3. On the NAME HERE dialog, click Enable to activate the packet capture feature.

Some details to be filled in there, but you get the idea.

Reply:
@vara2504